src/components/AudioPlayer.tsx

```diff
@@ -15,14 +16,19 @@ const StyledView = styled.View`
   margin-bottom: 20px;
 `
 
-const VolumeIcon = styled.TouchableOpacity`
+const VolumeIcon = styled.TouchableOpacity<{ disabled: boolean; isActive: boolean }>`
   position: absolute;
   top: -20px;
   left: 45%;
   width: 40px;
   height: 40px;
   border-radius: 50px;
-  background-color: ${props => props.theme.colors.lunesFunctionalIncorrectDark};
+  background-color: ${props =>
+    props.disabled
+      ? props.theme.colors.lunesBlackUltralight
+      : props.isActive
+      ? props.theme.colors.lunesRed
+      : props.theme.colors.lunesRedDark};
```

Comment:
You are doing it as well :astonished: Nested ternary expressions, tzzz :smile:

Reply:
Its not mine, i just copied :smile: